bb-email/src/main/java/com/cognifide/qa/bb/email/EmailClient.java

```diff
@@ -68,43 +67,44 @@ public void connect() {
 
   /**
    * @param subject Email message subject.
-   * @return List of Messages with subject from parameter.
+   * @return Array of Messages with subject from parameter.
    */
-  protected List<Message> getMessageList(String subject) {
+  protected Message[] getMessageList(String subject) {
```

Comment:
I prefer leaving Lists in the API as they offer more flexibility for the future.